### ISSUE TITLE
Add option to parametrize /metrics path

### ISSUE THREAD

**Issue by pnowy** [jochen-schweizer/express-prom-bundle#21]
Hi,

I think it would be good to have an option to parametrize metrics path during `promBundle` initialization. I know this is possible by the autoregister set to false and by adding own routing but in this case own routing path is added also to prometheus metrics.

Suggested configuration example:

```
const metricsMiddleware = promBundle({
  includeMethod: true,
  includePath: true,
  autoregister: true,
  metricsPath: '/prometheus' // instead of /metrics
})
```

**Comment by disjunction** [comment]
Hi @pnowy 

> but in this case own routing path is added also to prometheus metrics

it won't if you first register your custom route and then do `app.use(promBundle)`

Why I did not make it customizable is to discourage people from using own paths but use the default recommended one. Please let me know if you're still not convinced - i'll include it in the newer version (otherwise please close).

I need to do update the library soon due to #20 which is something I was planning to do anyway.

**Comment by pnowy** [comment]
Hi @disjunction,

thanks for the quick answer.

Yes - you are correct, I could register the metrics route before `app.use(promBundle)` and it works like a charm.

Anyway, from my perspective, it would be good to have this option directly on the options (as I've checked the code it should be a couple of lines in the code in the place where autoregister is checked).

In our case, the stack is built on top of the Java and applications are registered within Eureka (nodejs apps too) and we provide metrics to prometheus by eureka-consul-adapter. With this approach, we used different URL for prometheus than your default so that's why I think it is a good idea to have the option to parametrize it out-of-the-box.
